sdk/src/integrationTest/java/MirrorNodeCloseClientIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+import com.hedera.hashgraph.sdk.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.threeten.bp.Instant;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class MirrorNodeCloseClientIntegrationTest {
+    @Test
+    @DisplayName("Can close the Client while subscribed to mirror node topic queries")
+    void canCloseClient() throws Exception {
+        var testEnv = new IntegrationTestEnv(1);
```

Comment:
This test should not be an integration test. Tests that hit a remote system are brittle and should be avoided. There's already a stub-based test for `TopicMessageQuery` that should be enhanced for this scenario.